flixel/input/gamepad/id/FlxGamepadAnalog.hx

```diff
@@ -0,0 +1,26 @@
+package flixel.input.gamepad.id;
+
+import flixel.input.FlxInput.FlxInputState;
+import flixel.input.gamepad.FlxGamepad;
+import flixel.input.gamepad.FlxGamepadInputID;
+import flixel.input.gamepad.FlxGamepad.FlxGamepadAnalogStick;
+
+/**
+ * A helper class for gamepad input.
+ * Provides optimized gamepad button checking using direct array access.
+ * 
+ */
+@:keep
+class FlxGamepadAnalog
```

Comment:
Maybe this should remain to be called "FlxGamepadAnalogList" and value should be "FlxGamepadAnalogValueList"?